504.xml

```diff
@@ -52,123 +52,137 @@ https://www.omaspecworks.org/about/intellectual-property-rights/
 		<Mandatory>Optional</Mandatory>
 		<Resources>
 			<Item ID="0">
-				<Name>Current UIM Type</Name>
+				<Name>Current SIM Type</Name>
 				<Operations>R</Operations>
 				<MultipleInstances>Single</MultipleInstances>
 				<Mandatory>Mandatory</Mandatory>
 				<Type>Integer</Type>
 				<RangeEnumeration></RangeEnumeration>
 				<Units></Units>
 				<Description><![CDATA[Provides the information about the currently being used SIM Type:
-0: Full Size
-1: Mini Size
-2: Micro
-3: Nano
-4: eSIM
-5: UICC
-6: Soft SIM
-7: Thin SIM
-8: eUICC
-9: iSIM
-10: iUICC
-11-24: Reserved for future use]]></Description>
+0: UICC (removable)
+1: eUICC (removable)
+2: eUICC (non-removable)
+3: iUICC
+4-24: Reserved for future use]]></Description>
 			</Item>
 			<Item ID="1">
 				<Name>Supported SIM Type</Name>
 				<Operations>R</Operations>
-				<MultipleInstances>Single</MultipleInstances>
+				<MultipleInstances>Multiple</MultipleInstances>
 				<Mandatory>Mandatory</Mandatory>
 				<Type>Integer</Type>
 				<RangeEnumeration></RangeEnumeration>
 				<Units></Units>
 				<Description><![CDATA[Provides the information about the currently supported SIM Types: 
-0: Full Size
-1: Mini Size
-2: Micro
-3: Nano
-4: eSIM
-5: UICC
-6: Soft SIM
-7: Thin SIM
-8: eUICC
-9: iSIM
-10: iUICC
-11-24: Reserved for future use]]></Description>
+0: UICC (removable)
+1: eUICC (removable)
+2: eUICC (non-removable)
+3: iUICC
+4-24: Reserved for future use]]></Description>
 			</Item>
 			<Item ID="2">
-				<Name>Service Provide Name</Name>
+				<Name>Service Provider Name</Name>
 				<Operations>R</Operations>
 				<MultipleInstances>Single</MultipleInstances>
-				<Mandatory>Mandatory</Mandatory>
+				<Mandatory>Optional</Mandatory>
 				<Type>String</Type>
 				<RangeEnumeration></RangeEnumeration>
 				<Units></Units>
-				<Description><![CDATA[Provides the Service Provider Name using which LwM2M is currently communicating to the Server]]></Description>
+				<Description><![CDATA[Provides the name of the Service Provider assocatiated with the downloaded profile or the profile currently being downloaded.]]></Description>
 			</Item>
 			<Item ID="3">
 				<Name>Profile Package</Name>
 				<Operations>W</Operations>
 				<MultipleInstances>Single</MultipleInstances>
-				<Mandatory>Mandatory</Mandatory>
+				<Mandatory>Optional</Mandatory>
 				<Type>Opaque</Type>
 				<RangeEnumeration></RangeEnumeration>
 				<Units></Units>
 				<Description><![CDATA[Profile Package is a personalised Profile using an interoperable description format that is transmitted to an eUICC to load and install a Profile.]]></Description>
 			</Item>
 			<Item ID="4">
-				<Name>Profile URI</Name>
+				<Name>Download URI/Information</Name>
```

Comment:
How about "Profile or Download URI"?

Either the "Profile or Download URI" resource or the "Profile Package" resource must be defined. We need to capture that somewhere.